assets/models/system/automl-text-classification/spec.yaml

```diff
@@ -48,4 +47,4 @@ tags:
     apply_deepspeed: 'true'
```

Comment:
@yiyoulin can we verify these for nlp runtime? ignore if these are already verified...

Reply:
discussed offline for this.